README.md

```diff
@@ -32,6 +32,10 @@ Clone the repository
 
   - Production: `$ fvm flutter run --flavor production`
 
+- Generate assets folder
+  - `$ flutter pub get`
```

Comment:
I think this one is unnecessary 😄 we just need to run the below command

Reply:
@sleepylee Okay, I've removed it from README.